src/patternfly/demos/HelperText/examples/HelperText.md

```diff
@@ -0,0 +1,96 @@
+---
+id: 'Helper text'
+beta: true
+section: components
+---
+
+## Examples
+### Helper text on a form
+```hbs
+{{#> form form--id="helper-text-form"}}
+  {{#> form-group form-group--id="-name"}}
+    {{#> form-group-label}}
+      {{#> form-label form-label--attribute=(concat 'for="' form--id form-group--id '"') required="true"}}
+        Name
+      {{/form-label}}
+    {{/form-group-label}}
+    {{#> form-group-control}}
+      {{#> form-control controlType="input" input="true" form-control--attribute=(concat 'required type="text" id="' form--id form-group--id '" name="' form--id form-group--id '" aria-describedby="' form--id form-group--id '-helper"')}}
+      {{/form-control}}
+      {{#> form-helper-text form-helper-text--type="div" form-helper-text--attribute=(concat 'id="' form--id form-group--id '-helper" aria-live="polite"')}}
+          {{#> helper-text}}
```

Comment:
Looks like the `{{#> helper-text}}` block in these examples has an extra tab/indent.